source/PLL_API.h

```diff
@@ -0,0 +1,45 @@
+/** 
+ * @file        PLL_API.h
+ *
+ * @brief       PLL_API contains the PLL basic functions
+ *
+ * This Module implements the PLL basis functionalities. The Module contains
+ * functions to set the PLL output frequencys.
+ * 
+ * @author      Florian Heidecker
+ * @date        13.11.2015 - initial version
+ * 
+ * @version     0.1
+ *
+ */
+
+#ifndef _PLL_API_H
+#define _PLL_API_H
+
+#include <xc.h>
+
+/** avalible frequenz output on SCKO1 in MHz */
+typedef enum {_16 = 0, _33 = 1} SCKO1_FREQ;
+
+/** avalible sampling frequency in kHz */
+typedef enum {_48=0, _44_1=1, _32=2, _96=4, _88_2=5, _64=6} SAMPLING_FREQ;
```

Comment:
I am not sure if you should add a prefix to the elements in the enum. 
I think we will have similar enums in some other modules and the names will get a little bit confusing because they are not unique. 
We can discuss this tomorrow face to face.